src/components/molecules/AdminInput/AdminInput.module.scss

```diff
@@ -0,0 +1,4 @@
+.input {
+  padding: 0.5rem 0.75rem;
```

Comment:
Maybe we'll need to copy the old `font-size--root` and `font-size-parent` from the old `constants.scss` file or write similar ones so we can add some meaning to these 0.5 and 0.75...

But it should be for attendee side. Considering this is admin side, shouldn't Tailwind solve it in the `className` instead? 

I'd suggest we create an `Input` component in `./src/components/admin` that follows that style system.